protoc_plugin/lib/src/generated/descriptor.pb.dart

```diff
@@ -2226,7 +2226,7 @@ class MessageOptions extends $pb.GeneratedMessage {
   ///  maps field.
   ///
   ///  For maps fields:
-  ///      map<KeyType, ValueType> map_field = 1;
+  ///      `map<KeyType, ValueType> map_field = 1;`
```

Comment:
Is this intentional? This edits a generated file.

There are also other code blocks in this file that are not quoted like this one.

Reply:
It is. Perhaps the lint needs to be tweaked. It's to avoid treating `<` like HTML. We generally like triple-tick blocks for these, perhaps.

Reply:
I ignored this lint instead in generated code.